packages/libical.rb

```diff
@@ -0,0 +1,32 @@
+require 'package'
+
+class Libical < Package
+  description 'An open source reference implementation of the icalendar data type and serialization format'
+  version '3.0.8.99' # 3.0.8 doesn't compile.
+  compatibility 'all'
+  source_url 'https://github.com/libical/libical/archive/07dc84fb0d5e50fb493005e10fa41930111da48f.zip'
+  source_sha256 '5fdf39b0ea654589bf2e9aa88ef1b0206c18aa6987f5a4299db3748e8b1eefd6'
+
+  depends_on 'glib'
+  depends_on 'icu'
```

Comment:
Does this package exist?  I know `icu4c` does but since it's a core package it doesn't need listed anyway.

Reply:
Ah! changing this.